cmd/kyma/alpha/sign/module/module.go

```diff
@@ -0,0 +1,120 @@
+package module
+
+import (
+	"fmt"
+	"github.com/gardener/component-spec/bindings-go/ctf"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/pkg/module"
+	"github.com/mandelsoft/vfs/pkg/osfs"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *Options
+	cli.Command
+}
+
+//NewCmd creates a new Kyma CLI command
+func NewCmd(o *Options) *cobra.Command {
+
+	c := command{
+		Command: cli.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "module MODULE_NAME MODULE_VERSION [flags]",
+		Short: "Sign all module resources from unsigned component descriptor which is hosted in a remote OCI registry",
+		Long: `Use this command to sign a Kyma module.
+
+### Detailed description
+
+This command signs all module resources recursively based on an unsigned component descriptor hosted in an OCI registry with the provided private key. The output (component-descriptor.yaml), is then saved in the descriptor path (./mod as a default) as a signed component descriptor. If signed-registry are provided, it will push the signed component descriptor.
```

Comment:
What is "signed-registry" ? Is it singular or plural? Please advise.

```suggestion
This command signs all module resources recursively based on an unsigned component descriptor hosted in an OCI registry with the provided private key. Then, the output (component-descriptor.yaml) is saved in the descriptor path (default: ./mod) as a signed component descriptor. If signed-registry are provided, the command pushes the signed component descriptor.
```

Reply:
`signed-registry` is a flag value, the definition is here https://github.com/kyma-project/cli/pull/1319/files#diff-96dd0052d8c26963d4bc0702c9b39847a8103f8c7d2bfe498f9b3a9dd6d6bb52R42